### PR TITLE
Update CHANGELOG to graphql api names

### PR DESCRIPTION
**Pull request by jinjoolee07**
Related to: #805

### CODE REVIEW

**Comment by kimhanbeom** [comment]
@jinjoolee07 
- Replacing the method name with the GraphQL API name means replacing it with the name of the GraphQL API that is called externally.
- One example is that the actual function implementing qeury that provides dns traffic event in giganto is named `dns_raw_event`, but the actual GraphQL API call to it is named `dnsRawEvent`.
https://github.com/aicers/giganto/blob/301611dc2716c4ec54861da1c049e3cef9b017c4/src/graphql/network.rs#L1244-L1252
https://github.com/aicers/giganto/blob/301611dc2716c4ec54861da1c049e3cef9b017c4/src/graphql/client/schema/dns_raw_events.graphql#L2-L9

**Comment by sehkone** [comment]
@jinjoolee07 

According to our rules, we need to use GitHub keywords, such as "close", instead of manually linking the related issue.

File: CHANGELOG.md [hunk 1]
```diff
@@ -306,7 +305,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - `StorageKey`: A structure that stores lookup keys generated by `StorageKeyBuilder`.
   - `KeyExtractor`: A trait for calling the value to be set by the key.
 - Supports 14 sysmon events.
-- Added `search sysmon event` Graphql Api for `sysmon` supported by giganto.
+- Added `sysmonEvents` Graphql API for `sysmon` supported by giganto.
```
Comment (kimhanbeom) [inline CHANGELOG.md:308]: This part refers to queries of the search sysmon event (e.g. `searchProcessCreateEvents`), not the GraphQL API sysmonEvents,  Instead of reverting to the existing wording, it would be clearer to change it to "Added “search[sysmon type]Events" Graphql APIs for sysmon event supported by giganto".

**Review by kimhanbeom — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -129,10 +129,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
-- Added the ability to filter via `agent_id` in the filter of a GraphQL query
-  requesting a sysmon events.
-- Added `SmtpRawEvent` to the return value union of `network_raw_events` GraphQL
-  query.
+- Added the ability to filter via `agent_id` in the `sysmonEvents` GraphQL API.
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: 수정 완료했습니다

File: CHANGELOG.md [hunk 3]
```diff
@@ -411,7 +410,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add a publish API to return the source, raw_events
   from the source, timestamps for REconverge.
 - Supports the FTP protocol.
-- Add a GraphQL API to search http events
+- Add a GraphQL API to search `httpRawEvents`
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 4]
```diff
@@ -355,7 +354,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Supports the TLS protocol.
 - Added migration functionality in `0.12.0`. This feature adds values for new fields
   (`orig_filenames`, `orig_mime_types`, `resp_filenames`, `resp_mime_types`) to
-  `http raw event` in versions 0.12.0 and earlier.
+  `httpRawEvents` GraphQL type in versions 0.12.0 and earlier.
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 5]
```diff
@@ -320,7 +319,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   This change makes it possible to export statistics data of only core 0 of
   the collector device. This will be fixed in next change.
 - Remove unused `time()` from `RawEventFilter` trait.
-- Modify to use `Networkfilter` in `sysmon raw event` Graphql Api.
+- Modify to use `Networkfilter` in `sysmonEvents` Graphql API.
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md (hunk 1 above)
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 6]
```diff
@@ -355,7 +355,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Supports the TLS protocol.
 - Added migration functionality in `0.12.0`. This feature adds values for new fields
   (`orig_filenames`, `orig_mime_types`, `resp_filenames`, `resp_mime_types`) to
-  `http raw event` in versions 0.12.0 and earlier.
+  `Http` GraphQL type in versions 0.12.0 and earlier.
```
Comment (kimhanbeom) [inline CHANGELOG.md:358]: I apologize for any confusion. The migration part is not related to GraphQL, so please change the existing "http raw event" to "Http".

**Review by kimhanbeom — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 6 above)
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed. Thank you for the clarification, and apologies for the confusion earlier.

**Comment by sophie-cluml** [comment]
For consistency I'd like to suggest using "GraphQL", not graphQL, graphql, Graphql or any other casings.

**Comment by sophie-cluml** [comment]
@jinjoolee07 I think we need two PRs to completely resolve https://github.com/aicers/giganto/issues/805, because we need one toward main branch and  another toward tis branch. But I am afraid that this trigger phrase "Fixes: https://github.com/aicers/giganto/issues/805" will automatically mark the issue as complete, as soon as this PR gets merged. Could you take a look at it please?

**Comment by jinjoolee07** [comment]
> For consistency I'd like to suggest using "GraphQL", not graphQL, graphql, Graphql or any other casings.

Thank you for the suggestion! I’ve just made the edits to ensure it's consistently using 'GraphQL' throughout. Please let me know if I missed any instances.

**Review by kimhanbeom — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -257,7 +257,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Support `NetflowV5`, `NetflowV9` events. These events does not streamed
   to Hog or Crusher.
 - Support `Seculog` events.
-  - In graphql, `Seculog` requires its kind.
+  - The `Seculog` GraphQL API requires its kind.
```
Comment (sophie-cluml) [inline CHANGELOG.md:260]: `Seculog` 라는 GraphQL API endpoint는 없는 것 같은데 확인한 번 부탁드립니다.

**Comment by sophie-cluml** [comment]
@jinjoolee07 Could you check these lines please? Since below lines are not displayed in the "Files changed" tab, I could not make comments on the lines.
- https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR74 : For this, I think parenthesis should be removed.
- https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR145 : Maybe query to GraphQL API
- https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR150 :  Maybe query to GraphQL API
- https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR330 : I suggest "Fixed `export` GraphQL API to export statistics data for all cores."

File: CHANGELOG.md (hunk 7 above)
Comment (jinjoolee07) [reply CHANGELOG.md:260]: Seems that there is no direct API endpoint named Seculog

Comment (sophie-cluml) [reply CHANGELOG.md:260]: Could you check if that tried to mean `secu_log_raw_events` method?

Comment (jinjoolee07) [reply CHANGELOG.md:260]: It looks like there is no direct `Seculog` API endpoint. Instead, `Seculog` events are accessed through the `secu_log_raw_events` query, which retrieves events matching the `SecuLog` struct, including the `kind` field.

Comment (sophie-cluml) [reply CHANGELOG.md:260]: I think it is `secu_log_raw_events`, given that the changelog line was written in https://github.com/aicers/giganto/pull/575/files

**Comment by jinjoolee07** [comment]
> @jinjoolee07 Could you check these lines please? Since below lines are not displayed in the "Files changed" tab, I could not make comments on the lines.
> 
> * https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR74 : For this, I think parenthesis should be removed.
> * https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR145 : Maybe query to GraphQL API
> * https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR150 :  Maybe query to GraphQL API
> * https://github.com/aicers/giganto/pull/808/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR330 : I suggest "Fixed `export` GraphQL API to export statistics data for all cores."

Modification completed

File: CHANGELOG.md [hunk 8]
```diff
@@ -428,7 +429,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Add Giganto Restart processing.
-- Add more fields to `dns`, `conn`, `http`
+- Add more fields to `dns`, `conn`, `http`.
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 9]
```diff
@@ -411,7 +412,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add a publish API to return the source, raw_events
   from the source, timestamps for REconverge.
 - Supports the FTP protocol.
-- Add a GraphQL API to search http events
+- Add a GraphQL API to `searchHttpRawEvents`.
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 10]
```diff
@@ -17,11 +17,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - Fixed code and structures based on changes to the conn, http, smtp, ntlm,
     ssh, tls protocols field.
   - Update the compatibility version of the quic communication modules.
-    - Changed `PEER_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22."
-    - Changed `INGEST_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22."
-    - Changed `PUBLISH_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22."
+    - Changed `PEER_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22".
+    - Changed `INGEST_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22".
+    - Changed `PUBLISH_VERSION_REQ` to ">=0.21.0-alpha.1,<0.22".
```
[the remark this answers is not in the document]
Comment (jinjoolee07) [reply CHANGELOG.md]: Modification completed

File: CHANGELOG.md [hunk 11]
```diff
@@ -106,14 +106,14 @@ Versioning](https://semver.org/spec/v2.0.0.html).
...
-- Modify the `sources` query to return results that also include
+- Modify the `sources` GraphQL API to return results that also include
   sources from its peers.
 - Changed the prefix of the `Netflow5`/`Netflow9`/`SecuLog` db key to source.
 - Modify the related queries as the db key of `Netflow5`/`Netflow9`/`SecuLog`
   is changed to source.(`netflow5_raw_events`/`netflow9_raw_events`/
-  `secu_log_raw_events`)
-- Changed `PEER_VERSION_REQ` to ">=0.19.0,<0.20.0"
-- Changed `PUBLISH_VERSION_REQ` to ">=0.17.0,<0.20.0"
+  `secu_log_raw_events`).
```
Comment (sophie-cluml) [inline CHANGELOG.md:114]: I think this "queries" referred to GraphQL APIs. Could you check please?

**Comment by sophie-cluml** [comment]
Could you resolve conflict as well?

**Review by sophie-cluml — DISMISSED** [review]
LGTM

**Comment by sophie-cluml** [comment]
Since the codecov is not working as we intended in this PR, and I think https://github.com/aicers/giganto/issues/816 will resolve the problem. So I suggest to rebase this PR on top of main branch, after #816 is merged to the main. Then, I expect the codecov to compare this PR against the latest main commit, and thus pass the codecov check.

**Review by sophie-cluml — APPROVED** [review]
Thank you for the hard work.

**Comment by sophie-cluml** [comment]
@jinjoolee07 I am sorry but I would like to ask you for one more thing. Could you change "graphql api" to "GraphQL API" in your commit message?

**Comment by jinjoolee07** [comment]
> @jinjoolee07 I am sorry but I would like to ask you for one more thing. Could you change "graphql api" to "GraphQL API" in your commit message?

Modified the commit message to GraphQL API
